torchtext/csrc/bert_tokenizer.cpp

```diff
@@ -254,51 +266,56 @@ UString BERTEncoder::_basic_tokenize(const UString& text) {
 
 std::vector<std::string> BERTEncoder::Tokenize(std::string text) {
   std::vector<std::string> results;
+  std::vector<std::string> interim_results;
+  std::vector<std::string> tokens;
+  std::set<std::string> never_split(never_split_.begin(), never_split_.end());
```

Comment:
Does is make sense to create this set inside of the `Tokenize` function instead of the constructor? The set will be recreated every time this method is called which doesn't seem efficient.

Reply:
Good point! I will fix it.